src/faebryk/libs/sexp/dataclass_sexp.py

```diff
@@ -107,6 +109,7 @@ def _convert(
                 and len(args) == 2
```

Comment:
I was tempted to flatten this function into a single loop, since it's pretty complicated rn to follow, but started on it and realised there were enough corner cases that now wasn't the time to touch it